ts/torch_handler/utils/optimization/optimization.py

```diff
@@ -0,0 +1,19 @@
+from abc import abstractmethod

```

Comment:
Cool! I need to sanity check this with #1857 specifically some optimizations my modify state but what I like about this is it makes it clear that optimizations are composable